kahuna/public/js/edits/labeller.css

```diff
@@ -17,13 +19,18 @@
     background-color: #99ddff;
 }
 
-.label--partial,
-.label--partial gr-icon {
+/*
+label--partial is part of gr-panel.
+TODO refactor gr-panel to use common labeller component
+*/
+.label--partial .label__value,
```

Comment:
To win on specificity against https://github.com/guardian/grid/pull/1282/files#diff-493a18df09d616f3d63820529e14407cR33 ... I agree its not very nice.